api/proto/banyandb/measure/v1/query.proto

```diff
@@ -62,8 +62,20 @@ message QueryRequest {
   // tag_projection can be used to select tags of the data points in the response
   model.v1.TagProjection tag_projection = 5;
   message FieldProjection {
-    repeated string name = 1;
+    repeated string names = 1;
   }
   // field_projection can be used to select fields of the data points in the response
   FieldProjection field_projection = 6;
+  message GroupBy {
+    model.v1.TagProjection tag_projection = 1;
+    string field_name = 2;
+  }
+  // group_by groups data points based on their field value for a specific tag and use field_name as the projection name
+  GroupBy group_by = 7;
+  message Aggregation {
+    model.v1.AggregationFunction function = 1;
+    string field_name = 2; 
```

Comment:
To be specific, please be clear about the rule, should this `field_name` be in `field_projection`? This is sensitive from user perspective.